example-app-python-django/docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+services:
+
+  roach1:
+    container_name: roach1
+    hostname: roach1
+    image: cockroachdb/cockroach:v22.2.2
+    networks:
+      - roachnet
+    ports:
+      - "26257:26257"
+      - "8080:8080"
+    command: start --insecure --join=roach1,roach2,roach3
+    volumes:
+      - ~/cockroach-data/roach1:/cockroach/cockroach-data
+  
+  roach2:
+    container_name: roach2
+    hostname: roach2
+    image: cockroachdb/cockroach:v22.2.2
+    networks:
+      - roachnet
+    command: start --insecure --join=roach1,roach2,roach3
+    volumes:
+      - ~/cockroach-data/roach2:/cockroach/cockroach-data
+    depends_on:
+      - roach1
+  
+  roach3:
+    container_name: roach3
+    hostname: roach3
+    networks:
+      - roachnet
+    image: cockroachdb/cockroach:v22.2.2
+    command: start --insecure --join=roach1,roach2,roach3
+    volumes:
+      - ~/cockroach-data/roach3:/cockroach/cockroach-data
+    depends_on:
+      - roach1
+
+  weba:

```

Comment:
Add an environment variable that specifies DB port. Set it to `26257`, so that it connects to roach1